gems/yrb-actioncable/.tool-versions

```diff
@@ -1 +1,4 @@
 redis 6.2.7
+ruby 3.2.0
+nodejs 19.6.0
```

Comment:
I wasn't totally sure if we needed to pin node as well to get `npm` or if adding `yarn` is sufficient :thinking:

Reply:
This is fine. We should pin the Node.js version in `.nvmrc` and the Ruby one in `.ruby-version` to ensure things keep working for non `asdf` users.

Reply:
OK :+1: 

btw I noticed that Turborepo says you should choose either npm, yarn or pnpm. Do we actually need both? :thinking: 

https://turbo.build/repo/docs/getting-started/create-new#which-package-manager-do-you-want-to-use